packages/appium/docs/en/guides/execute-methods.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Execute Methods
+---
+
+Because the scope of commands implemented in Appium drivers is broader than the scope of commands
+defined by the W3C WebDriver spec, Appium needs a way for these "extended" commands to be accessed
+by client libraries. There are two main strategies for this:
+
+1. Appium drivers define new W3C-compatible API routes, and Appium clients are updated to include
+   support for those new routes.
+2. Appium drivers define so-called "Execute Methods" which provide access to new functionality by
+   overloading the existing `Execute Script` command which is already available in any WebDriver-
+   based client library (including all Selenium and Appium clients).
+
+There are pros and cons for each strategy, but it is ultimately up to a driver or plugin author to
+decide how they wish to provide access to any new commands. You should always refer to the
+documentation for a specific driver or plugin to see how it expects commands to be called.
```

Comment:
```suggestion
decide how they wish implement new commands. 
```

that last sentence seems out-of-place, as this doc looks to be information for extension authors, not end-users